workflow/build_opts.yaml

```diff
@@ -0,0 +1,90 @@
+systems:
+  common:
+    - "ufs_utils"
+    - "gfs_utils"
+    - "upp"
+  gfs:
+    - "gfs_model"
+    - "gfs_ww3prepost"
+    - "gsi_enkf"
+    - "gdas"
+    - "gsi_utils"
+    - "gsi_monitor"
```

Comment:
We could probably offer more systems here:
`gfs` - as is here (this would cover the JEDI and GSI cycled options as well as the forecast only option)
`gfs_forecast_only` - just `gfs_model` and `gfs_ww3prepost`
`gfs_gsi` - everything here except `gdas`

Reply:
Good idea, added.